scripts/entry/app.py

```diff
@@ -0,0 +1,97 @@
+from dataclasses import dataclass
+from typing import Dict
+import cfn_flip
+import json
+import subprocess
+import pathlib
+
+class Utils:
+    @staticmethod
+    def get_dict_from_template(filepath):
+        with open(filepath) as f:
+            template_raw = f.read()
+        template_flat_json = cfn_flip.to_json(template_raw)
+        template_dict = json.loads(template_flat_json)
+        return template_dict
+
+@dataclass
+class EntryTemplateFactory:
+    boilerplate: Dict
+    main:        Dict
+    entry:       Dict = None
```

Comment:
should use native `dict` type now (python 3.9+)
```suggestion
    boilerplate: dict
    main:        dict
    entry:       dict = None
```

Reply:
Hmm, astute!